num2words/lang_RU.py

```diff
@@ -134,7 +155,42 @@ def pluralize(self, n, forms):
         return forms[form]
 
     def to_ordinal(self, number):
-        raise NotImplementedError()
+        self.verify_ordinal(number)
+        outwords = self.to_cardinal(number).split(" ")
+        lastword = outwords[-1].lower()
+        print(outwords)
+        try:
+            if len(outwords) > 1:
+                if outwords[-2] in self.ords_feminine:
+                    outwords[-2] = self.ords_feminine.get(
+                        outwords[-2], outwords[-2])
+            if len(outwords) == 3:
+                if outwords[-3] in ['один', 'одна']:
+                    outwords[-3] = ''
+            lastword = self.ords[lastword]
+            print(lastword)
+        except KeyError:
+            if lastword[:-3] in self.ords_feminine:
+                print(lastword)
+                lastword = self.ords_feminine.get(
+                    lastword[:-3], lastword) + "сотый"
+            elif lastword[-1] == "ь" or lastword[-2] == "т":
+                lastword = lastword[:-1] + "ый"
+            elif lastword[-1] == "к":
+                lastword = lastword + "овой"
+            elif lastword[-5:] == "десят":
+                lastword = lastword.replace('ь', 'и') + 'ый'
+            elif lastword[-2] == "ч" or lastword[-1] == "ч":
+                if lastword[-2] == "ч":
+                    lastword = lastword[:-1] + "ный"
+                if lastword[-1] == "ч":
+                    lastword = lastword + "ный"
+            elif lastword[-1] == "н" or lastword[-2] == "н":
+                lastword = lastword[:lastword.rfind('н') + 1] + "ный"
+            elif lastword[-1] == "д" or lastword[-2] == "д":
+                lastword = lastword[:lastword.rfind('д') + 1] + "ный"
+        outwords[-1] = self.title(lastword)
+        return " ".join(outwords).strip()

```

Comment:
@smirnovvad  There are some `print()` statements that it seems you used to debug. Are they needed? If not Could you remove them?

Reply:
Forgot about it, sorry :)

Reply:
Thanks!